cylc/uiserver/handlers.py

```diff
@@ -122,7 +78,6 @@ def set_default_headers(self) -> None:
         self.set_header("Content-Type", 'application/json')
 
     @web.authenticated
-    @authorised
```

Comment:
This endpoint should be authorised.

The information it contains is not especially sensitive, especially given the authenticated requirement, however, it's still data you wouldn't expect another user to be able to see.

Reply:
Authorization now added to Main handler and UserProfile handler - https://github.com/cylc/cylc-uiserver/pull/204/commits/baa081a35c58b7716a386c7ecbb3a8934cb8403c
These are now authorized on a `read` access mutation basis (configurable on a per user basis with the same `site_authorization` and `user_authorization` as used in the Middleware).  @kinow, not sure if you want to have a glance at this commit or if you are happy leaving it to @dwsutherland and @oliver-sanders.